### ISSUE TITLE
Firefox version detected partially.

### ISSUE THREAD

**Issue by vaanis** [bestiejs/platform.js#188]
My Firefox browser version is 81.0.1
but platform.js is detecting only 81.0

Is there a way for platform.js to fetch the complete version i.e., 81.0.1?
we are using an older version of platform.js from 2018 but even when I updated to the latest platform.js this issue persists.

Attached two files with the browser versions.
Browser version coming from platform.js:
![browser_version_platformJS](https://user-images.githubusercontent.com/70228328/95514657-287a7300-0971-11eb-874a-e1aadb5bb5d4.PNG)
------------------------------------------------
browser version from Firefox:
![Firefox_version](https://user-images.githubusercontent.com/70228328/95514665-2adccd00-0971-11eb-8f0b-877d56cc34ca.PNG)

**Comment by basvandenwijngaard** [comment]
I came across the same issue. So I checked the User Agent string, but Firefox doesn't pass the patch-part of the version number in it's User Agent string, only the major and minor parts are included.

For example: When you are on Firefox 81.0.1, only 81.0 is passed by Firefox in the User Agent string.

It seems to me that it is impossible to fetch the full version number, if the browser is not telling it's full version number.
